### PR TITLE
Hack on dns

### PR DESCRIPTION
**Pull request by okoreniuk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     <groupId>com.couchbase.client</groupId>
     <artifactId>java-client</artifactId>
-    <version>2.6.1</version>
+    <version>2.6.1-goeuro</version>
```
Comment (dnault) [inline pom.xml]: All the POM changes should probably be reverted.

**Comment by dizzzz** [comment]
what is this PR about?

**Comment by subalakr** [comment]
Hi @okoreniuk,
Is there a specific issue you are running into with not doing reverse address lookups? 
Currently, we know that encryption library being optional doesn't work well with the Spring environment, by chance are you running into the same problem else can you describe the environment you have and the specific class not exception you are running into.

**Comment by ingenthr** [comment]
Completed over on review.couchbase.org.  Thanks!

**Pull request closed without merging**